news/2018-04-23-twc-quarters-recap.md

```diff
@@ -0,0 +1,114 @@
+
+
+The third and final upset of the weekend, Spain versus Indonesia, was a real thriller. After trading close points with each other for 6 maps, the two teams ended up on tiebreaker where **Spain clutched out the victory.** 
+![](/wiki/shared/news/2018-04-19-twc-ro16-recap/spain-indo-tiebreaker.jpg)
+
+
+For the Round Of 16, this meant that Indonesia would face their fellow top seed France while Spain faced the United Kingdom. Both of those were rematches from last year where France and Spain came out on top, so revenge-driven performances from Indonesia and United Kingdom were to be expected.
+
+
+### Group Stage Mappool
+
+**The nomod picks were the clear favourites for the teams in Groups this year.** One of the most picked maps was Firce777's classic memorization map [Yuitsuki yukari - Shiawase ni nareru kakushi komando ga aru rashii](https://osu.ppy.sh/beatmapsets/219733/#taiko/514465), which is a type of map that has not been too popular historically. Instead, the outlier in nomod was the accuracy map [Chroma - Collapse](https://osu.ppy.sh/beatmapsets/691352#taiko/1463029) which surprisingly only got picked twice. 
+
+The rejection of accuracy maps in this stage was made even more apparent by the bans. The OD7 Hidden map [Ice - Entrance](https://osu.ppy.sh/beatmapsets/755569#taiko/1590333) and the long DoubleTime pick [FELT - Day After](https://osu.ppy.sh/beatmapsets/323180#taiko/718430) combined made up almost half of all Group Stage bans. 
+
+A word on the picks/bans by [Nwolf](https://osu.ppy.sh/users/1910766), one of the map selectors: 
+
+*We were surprised about the huge amount of picks on Shiawase and Goin' Under which we thought would be banned and the high amount of bans on Entrance, which we considered the easier HD map.* 
+
+
+### Round Of 16 Matches
+
+Unlike in the Group Stage, **the Round Of 16 didn't give us any close scorelines.** The 2nd place teams unfortunately only managed to steal 3 points in total from the group winners.
+
+**Remember those upsets I mentioned?** Brazil's loss to South Korea in the Group Stage meant they faced the titans that are the United States. The Brazilians were a couple players short and couldn't get a point on the board against the ruthless accuracies from the US. 
+
+Canada made quick work of Malaysia with a 5-0 victory, and Taiwan did the same against top-seeded Australia, forcing the Maustraysian duo into the losers bracket along with Brazil.
+
+Things got really interesting in the match between France and Indonesia. The losses of France usually came down to a single misplay, and they eventually took a point on the FreeMod [t+pazolite - Elder Dragon Legend (ft. Kabocha)](https://osu.ppy.sh/beatmapsets/761951#taiko/1602056) despite the 2 mod advantage of Indonesia. Overall though, **Indonesia will take their revenge on France 5-1**, the same score they lost by last year.
+
+Only one Argentinian player woke up for their match, resulting in a **win by default for Japan.**
```

Comment:
"woke up for their match" - change this for "made it to their match" so that it sounds a little more neutral